README.md

```diff
@@ -178,7 +166,7 @@ Feel free to check the [issues page](../../issues/).
 
 ## ⭐️ Show your support <a name="support"></a>
 
-If you like this project...
+If you like this project kindly consider giving it a star⭐
 
```

Comment:
Great job @flemton 🤝

- [ ] [Optional] Kindly fix the broken `MIT license` link and the `issues` link to make your readme file more professional.